packages/components/ng-ovh-utils/src/srefScrollTop/srefScrollTop.js

```diff
@@ -0,0 +1,28 @@
+import angular from 'angular';
+
+const moduleName = 'ua.srefScrollTop';
+
+angular
+  .module(moduleName, ['ui.router'])
+  .run(
+    /* @ngInject */ ($transitions, $document, $timeout, $anchorScroll) => {
+      $transitions.onSuccess(
+        {
+          entering: (state, transition) =>
+            transition.options().custom?.scrollTop,
+        },
+        () => {
+          $document[0].body.style.setProperty('height', 'auto', 'important');
+          $document[0].body.style.setProperty('overflow', 'auto');
+          $anchorScroll();
+          $timeout(function() {
+            $document[0].body.style.removeProperty('height');
+            $document[0].body.style.removeProperty('overflow');
+          });
```

Comment:
I think there exist an argument for directly scroll to top with $anchorScroll(); without change the style.

Reply:
There is only yOffset argument for $anchorScroll()